app/src/main/java/com/example/kuit6_week1/MainActivity.kt

```diff
@@ -19,29 +37,159 @@ class MainActivity : ComponentActivity() {
+                        color = Color.Black,
+                        shape = CircleShape
+                    )
+            )
+        }
+
+        SampleRow (
+            modifier = Modifier.padding(top = 20.dp)
+        ) {
+            Box(
+                modifier = modifier
+                    .size(size = 76.dp)
+                    .border(
+                        width = 1.dp,
+                        color = Color.Black
+                    )
+                    .padding(all = 8.dp)
+                    .background(Color.Red)
+            )
+
+            Box(
+                modifier = modifier
+                    .size(size = 76.dp)
+                    .border(
+                        width = 1.dp,
+                        color = Color.Black
+                    )
+                    .padding(all = 8.dp)
+                    .clip(CircleShape)
+                    .background(Color.Blue)
+            )
+        }
+
+        SampleRow(
+            modifier = Modifier.padding(all = 20.dp)
+        ) {
+            Image(
+                painter = painterResource(id = R.drawable.image),
+                contentDescription = "개 이미지",
+                modifier = Modifier.size(size = 80.dp)
+            )
+
+            Image(
+                painter = painterResource(id = R.drawable.image),
+                contentDescription = "개 이미지",
+                modifier = Modifier
+                    .size(size = 80.dp)
+                    .clip(CircleShape)
+            )
+        }
+
+        SampleRow(
+            modifier = Modifier.padding(all = 10.dp)
+        ) {
+            Column (
+                modifier = modifier
+                    .size(240.dp,140.dp)
+                    .border(
+                        1.dp,
+                        color = Color.Black,
+                        shape = RoundedCornerShape(size = 10.dp)
+                    )
+            ){
+                Column(
+                    modifier = Modifier.padding(16.dp),
+                            verticalArrangement = Arrangement.spacedBy(3.dp)
+                ) {
+                    Row(
+                        modifier = Modifier.fillMaxWidth(),
+                        horizontalArrangement = Arrangement.End
+                    ){
+                        Box{
+                            Image(
+                                painter = painterResource(id = R.drawable.image2),
+                                contentDescription = "쿠잇 이미지",
+                                modifier = Modifier
+                                    .size(size = 40.dp)
+                                    .clip(RoundedCornerShape(size = 10.dp))
+                            )
+                        }
+                    }
+
+                    Column(
+                        verticalArrangement = Arrangement.spacedBy(3.dp)
+                    ) {
+                        Divider(color = Color.Black, thickness = 0.5.dp)
```

Comment:
컴포즈 라이브러리에서 더이상 Divider가 권장되지 않아서 이런 식으로 작성하심 경고가 발생할 수 있습니다 ! 하단에 작성해드린 방식으로 수정해주세요 ~
```suggestion
                       HorizontalDivider(color = Color.Black, thickness = 0.5.dp)
```